flexmeasures/data/models/planning/storage.py

```diff
@@ -472,6 +480,33 @@ def _prepare(self, skip_validation: bool = False) -> tuple:  # noqa: C901
             max_value=ems_power_capacity_in_mw,
         )
 
+        if "ems_consumption_capacity_in_mw" in self.flex_context:
```

Comment:
`"ems_consumption_capacity_in_mw"`  but in line 488 it's `"ems_soft_consumption_capacity_in_mw"`?

Reply:
right... I'll fix that.